doc/source/cluster/kubernetes/getting-started.ipynb

```diff
@@ -244,6 +258,163 @@
     "Note that in production scenarios, you will want to use larger Ray pods. In fact, it is advantageous to size each Ray pod to take up an entire Kubernetes node. See the [configuration guide](kuberay-config) for more details."
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "ef55f61f",
+   "metadata": {},
+   "source": [
+    "### Deploying a static Ray cluster\n",
+    "\n",
+    "\n",
+    "#### Deploying a static Ray cluster without KubeRay\n",
+    "\n",
+    "In this section, we will deploy a static Ray cluster into the default namespace without using KubeRay. To use a non-default namespace, specify the namespace in your kubectl commands:\n",
+    "\n",
+    "`kubectl -n <your-namespace> ...`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3a8267f1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Deploy a sample Ray Cluster from the Ray repo:\n",
+    "! kubectl apply -f https://raw.githubusercontent.com/ray-project/ray/master/doc/source/cluster/kubernetes/configs/static-ray-cluster.with-fault-tolerance.yaml\n",
+    "\n",
+    "# Note that the Ray cluster has fault tolerance enabled by default using the external Redis. Please set the Redis IP address in the config."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "134f6e63",
+   "metadata": {},
+   "source": [
+    "Once the Ray cluster has been deployed, you can view the pods for the head node and worker nodes by running"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e60e518b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! kubectl get pods\n",
+    "\n",
+    "# NAME                                             READY   STATUS    RESTARTS   AGE\n",
+    "# deployment-ray-head-xxxxx                        1/1     Running   0          XXs\n",
+    "# deployment-ray-worker-xxxxx                      1/1     Running   0          XXs\n",
+    "# deployment-ray-worker-xxxxx                      1/1     Running   0          XXs"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "cd8e59c6",
+   "metadata": {},
+   "source": [
+    "Wait for the pods to reach Running state. This may take a few minutes -- most of this time is spent downloading the Ray images. In a separate shell, you may wish to observe the pods' status in real-time with the following command:"
```

Comment:
```suggestion
    "Wait for the pods to reach the `Running` state. This may take a few minutes -- most of this time is spent downloading the Ray images. In a separate shell, you may wish to observe the pods' status in real-time with the following command:"
```